cloudslang_build_tool.rst

```diff
@@ -192,9 +193,15 @@ following arguments:
 +------------+--------------------------+-------------------------------------------------------+
 | -cov       | off                      | whether or not test coverage data should be output    |
 +------------+--------------------------+-------------------------------------------------------+
-| -des       | off                      | | whether or not to validate description data against |
-|            |                          | | actual inputs, outputs and results                  |
+| -des       | off                      | | whether or not to validate that all inputs, outputs |
+|            |                          | | and results have descriptions                       |
 +------------+--------------------------+-------------------------------------------------------+
+| -par       | false                    | whether or not parallel test execution should be used |
```

Comment:
we also have a paremeter:
```
    @Parameter(names = {"--thread-count", "-th"},
            description = "Number of threads to be used in case of parallel test execution. " +
                    "Has no effect for sequential execution. By default, it is set to the number of processors.")
    public String threadCount;
```